File: frontend/src/features/actividades/rectificatoria_postventa_solucion_reparo/models/rectificatoria_postventa_solucion_reparo.ts
export interface RectificatoriaPostventaSolucionReparo {
    id_rectificatoria_postventa_solucion_reparo: number;
    id_expediente: number;
    id_usuario_solicitante: number;
    is_subsanar: boolean;
    observaciones?: string | null;
    modificar_datos_memo: boolean;
    descontabilizar_operacion: boolean;
    /**
     * Campos heredados/visualización.
     * Vienen desde el reparo generado en rectificatoria_analisis_derivacion_reparo_postventa
     * y/o desde la consulta con Users.
     */
    solicitante?: string | null;
    observaciones_reparo?: string | null;
    fecha_ingreso?: string | null;

    is_active: boolean;
    row_status: boolean;
    created_by: number;
    created_date: string;
    modified_by?: number | null;
    modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/rectificatoria_postventa_solucion_reparo.cs
using Common.Domain.Models;

namespace Multibanca.Domain.Models.Multibanca
{
    public class rectificatoria_postventa_solucion_reparo : base_auditoria
    {
        public int id_rectificatoria_postventa_solucion_reparo { get; set; }

        public long id_expediente { get; set; }

        public int id_usuario_solicitante { get; set; }

        public bool is_subsanar { get; set; }
        public bool modificar_datos_memo { get; set; }
        public bool descontabilizar_operacion { get; set; }

        public string? observaciones { get; set; }

        public string? solicitante { get; set; }

        public string? observaciones_reparo { get; set; }

        public DateTime? fecha_ingreso { get; set; }
    }
}
